spring-vault-core/src/main/java/org/springframework/vault/core/VaultTransitOperations.java

```diff
@@ -222,4 +222,67 @@
 	 * @see #rotate(String)
 	 */
 	String rewrap(String keyName, String ciphertext, VaultTransitContext transitContext);
+
+	/**
+	 * Generate HMAC digest of given data.
+	 *
+	 * @param keyName must not be empty or {@literal null}.
+	 * @param plainText must not be empty or {@literal null}.
+	 * @return the digest of given data using sha2-256 hash algorithm and the named key.
+	 */
+	String generateHmac(String keyName, String plainText);
```

Comment:
Let's accept `Plaintext` as type here instead of a plain string.

Reply:
Hi Marc,
Thanks a lot for your feedback. Shall I overload the method with a Plaintext argument or change this one to only accept Plaintext?

Reply:
Let's just accept `Plaintext` instead of `String`. Having an overload is tempting but we would end up with three `generateHmac` methods. Same for the other new methods.

Reply:
Does it make sense here to return `Hmac` object as well?

Reply:
I hadn't a clear idea on this one yet. Considering the development, it would make sense to return a `Hmac` type. This would give us some consistency with the other methods.